### ISSUE TITLE
Bug when get PagedList workflow-definition (Server API Endpoint)

### ISSUE THREAD

**Issue by dohoangtung** [elsa-workflows/elsa-core#2732]
Hello,
I'm working with Server Endpoints and found this issue:
I have total 5 workflows in database,

When I get data with no pageSize passed  -> server return all 5 workflows
![image](https://user-images.githubusercontent.com/40049759/152761907-72d087b6-6173-4e3e-9de4-2a351b731318.png)

then I've passed pageSize: 

![image](https://user-images.githubusercontent.com/40049759/152761693-ee4feb53-5dd8-4b57-a232-2a5ed1a62d69.png)

system return wrong data

**Comment by dohoangtung** [comment]
Oh I've realized that page start with 0 not 1

**Comment by sfmskywalker** [comment]
Indeed, the API uses zero -indexed pagination. We should probably include that in the Open API documentation.
